types/tryghost__content-api/index.d.ts

```diff
@@ -21,6 +22,11 @@ export interface Identification {
     id: string;
 }
 
+export interface ResourceIdentification {
```

Comment:
This type allows for an empty object to be provided, which I assume will not work.
This would work better as `{ slug: Nullable<string> } | { id: Nullable<string> }`.

Reply:
Dear @uniqueiniquity, this case is covered in the original code here:
```
           if (!data.id && !data.slug) {
                return Promise.reject(new Error('Must include either data.id or data.slug'));
            }
```
Nevertheless, I find your suggestion a much clearer way to avoid this happening in the first place.
Updated the code accordingly.